### PR TITLE
Workingleds

### PR DESCRIPTION
**Pull request by jbko6**

### CODE REVIEW

File: src/main/java/frc/team2412/robot/Robot.java [hunk 1]
```diff
@@ -157,6 +157,9 @@ public void autonomousInit() {
 			subsystems.drivebaseSubsystem.resetGyroAngleWithOrientation(Rotation2d.fromDegrees(180));
 			autonomousChooser.getAuto().schedule();
 		}
+		if (subsystems.armLedSubsystem != null) {
+			subsystems.armLedSubsystem.setLEDAutonomous();
+    }
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/Robot.java:162]:
```suggestion
		}
```

File: src/main/java/frc/team2412/robot/Robot.java [hunk 2]
```diff
@@ -167,6 +170,10 @@ public void autonomousInit() {
 	@Override
 	public void teleopInit() {
 		Shuffleboard.startRecording();
+
+		if (subsystems.armLedSubsystem != null) {
+			subsystems.armLedSubsystem.setLEDAlliance();
+    }
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/Robot.java:176]:
```suggestion
		}
```

**Review by jamesdooley4 — APPROVED** [review]

File: src/main/java/frc/team2412/robot/subsystems/ArmLEDSubsystem.java [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+							enableLED(true);
+							setLEDAlliance();
+						}));
+
+		armLEDTab = Shuffleboard.getTab("Arm LED");
+
+		armLEDTab.add("Color 1", color1Chooser).withPosition(0, 0).withSize(2, 1);
+		armLEDTab.add("Color 2", color2Chooser).withPosition(2, 0).withSize(2, 1);
+		armLEDTab.add("Color 3", color3Chooser).withPosition(4, 0).withSize(2, 1);
+		armLEDTab.add("State", stateChooser).withPosition(6, 0).withSize(2, 1);
```
Comment (KangarooKoala) [inline src/main/java/frc/team2412/robot/subsystems/ArmLEDSubsystem.java:170]: I don't think we actually use the selected value of `color3Chooser` or `stateChooser`, and I think this was behavior we didn't test yesterday. (We do use `color3Chooser` to set `color3` once but `color3` isn't used either)

Comment (jbko6) [reply src/main/java/frc/team2412/robot/subsystems/ArmLEDSubsystem.java:170]: Yeah, the choosers don't work currently as we saw yesterday. Will need to be addressed in a future pr. But right now we at least do have the alliance colors during teleop and green during auto

Comment (jbko6) [reply src/main/java/frc/team2412/robot/subsystems/ArmLEDSubsystem.java:170]: For the choosers to work we have to monitor NT for a change in value of the selection and then run the instant command that was selected

**Pull request merged**